ui/src/store/modules/sharedDimensions.ts

```diff
@@ -8,11 +8,13 @@ import { Collection, RdfResource } from 'alcaeus'
 export interface SharedDimensionsState {
   entrypoint: null | RdfResource
   collection: null | Collection,
+  hierarchies: null | Collection,
```

Comment:
We could create a separate store for hierarchies. But I guess that's also ok.

Reply:
Thought about that but ultimately we're querying all from same Lindas so maybe this is more similar

Reply:
Ah wait, this is not about the database :)

But conversely, maybe this reflects that dimensions and hierarchies indeed live closely as resources 🤔

Reply:
I'm fine either way. We may extract a separate store if hierarchies become a larger concern in the app.